dev-haskell/hspec-megaparsec/hspec-megaparsec-2.2.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+#hackport: flags: -dev
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Utility functions for testing Megaparsec parsers with Hspec"
+HOMEPAGE="https://github.com/mrkkrp/hspec-megaparsec"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=dev-haskell/hspec-expectations-0.8:=[profile?] <dev-haskell/hspec-expectations-0.9:=[profile?]
+	>=dev-haskell/megaparsec-9.0:=[profile?] <dev-haskell/megaparsec-10.0:=[profile?]
```

Comment:
Hm, again i thought I resolved all the megaparsec revdeps, and then bumped megaparsec.

It's completely possible that my git-fu has failed me, especially since I did some of these rebases on my phone 😯

I'll take a close look at the reflex and megaparsec stuff, i want to have it all sorted in this PR

Reply:
Ah, I might have grepped incorrectly. Deleting outdated unused ebuilds (if feasible) should make it clearer as well.

Reply:
You know, I think this is one of those situations where we just have to pick the least painful option:

1. older `hspec-megaparsec` does not allow `megaparsec-9.0.1`, therefore we should bump `hspec-megaparsec` first
2. newer `hspec-megaparsec` depends on a non-existent `>=megaparsec-0.9`, so we should bump `megaparsec` first

I think I'll bump `hspec-megaparsec` first, but let me know if your preference is different.